approval_polls/static/approval_polls/create.js

```diff
@@ -18,4 +18,35 @@ $(function () {
 
   $('[data-toggle=tooltip]').tooltip();
   $('button#add-choice').click($.proxy(this.addChoiceField, this));
+
```

Comment:
Could you add a comment to explain the purpose of the following code concerning the datetimepicker ? Just to make a separation between 'choices' and the 'datetimepicker' functionality.